client/state/atomic-transfer-with-plugin/selectors/index.ts

```diff
@@ -1 +1,10 @@
-// Get transfer and plugin status
+import { get } from 'lodash';
+import type { AppState } from 'calypso/types';
+
+export function getAtomicInstallStatus( state: AppState, siteId: number | null ): any {
+	return get( state, [ 'softwareInstall', siteId ?? 0 ], {} );
```

Comment:
It looks get `get` is commonly used in selectors but we're trying to deprecate lodash in Calypso: p4TIVU-9Bf-p2 Is it worth replacing with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) or other checks?

Reply:
I thought I heard some noise about that. Will update.